integration_tests/e2e/searchPrisoner.cy.ts

```diff
@@ -60,7 +64,10 @@ context('Contacts', () => {
       searchPrisonerPage.prisonerSearchFormField().clear().type(prisonerNumber)
       searchPrisonerPage.prisonerSearchSearchButton().click()
 
-      Page.verifyOnPage(SearchPrisonerPage).clicktoViewContactsList()
+      Page.verifyOnPage(SearchPrisonerPage).clickToViewContactsList()
+
+      const listContactsPage = new ListPrisonerContactsPage('Smith, John')
+      listContactsPage.checkOnPage()
     })
```

Comment:
I renamed the searchPrisoner test to manageContacts.... and was building up the journey tests there.